### DISCUSSION TITLE
Slash Commands Vanishing

### DISCUSSION THREAD

**Discussion by Mehul343** [nextcord/nextcord#450] [Q&A] [answered]
Slash Commands vanish after first use of a slash command. I am using guild_ids to add commands to a server. When I start the bot I can use any of the commands but after the first use of a command all the commands vanish and don't appear anymore when you type / in the chatbox.

**Comment by MaskDuck** [comment, marked as answer]
It sounds weird. Can you provide a video (if you can) and your code?

**Reply by LeSpatiocorne** [reply]
Have you the last update of nextcord ? I was on nextcord 2.0.0a6, some my slash commands vanished and some of them were issuing errors, after the update in 2.0.0a8 no more problem.

**Reply by Mehul343** [reply]
It is fixed now ty

**Comment by ooliver1** [comment]
For others that may have this issue, it's sometimes caused by multiple running instances of your bot - you can test this if normal text commands reply twice
